test

```diff
@@ -13,24 +15,66 @@ function test()
     local harness="$1"
     local mode="$2"
 
+    setup "$harness" "$mode"
+    setup_dynamic_mountpoint
+    install_environment
+    check_environment_started "$harness" "$mode"
+    run_pipeline_tests
+    restart_environment
+    check_environment_started "$harness" "$mode"
+    teardown
```

Comment:
Should this be inside if mode == static?

Reply:
It handles the default version of dynamic too (mountpoints on linux)

Reply:
aha, okay

Reply:
I'll add a comment to explain :)